.github/workflows/ci_tdd.yml

```diff
@@ -17,7 +17,18 @@ jobs:
           python-version: "3.8"
 
       - name: make the checkout in a sane way.
-        run: git clone https://github.com/${{ github.event.pull_request.user.login }}/galaxy_ng; cd galaxy_ng ; git checkout "${{ github.head_ref }}"
```

Comment:
why not just check out base_ref instead of head_ref, and compare against that?

I'm pretty much doing the same in https://github.com/ansible/ansible-hub-ui/blob/master/.github/workflows/pr-checks.yml#L85-L94

Reply:
You are probably right, but the issue we found with this current approach was that the forked repo's master branch was multiple commits behind upstream's master and the diff was showing a lot of files that weren't actually changed by the PR. So I'm taking a different approach all together and just curl'ing the PR's diff and parsing that.